tools/diff/macros.xml

```diff
@@ -0,0 +1,5 @@
+<macros>
+    <token name="@TOOL_VERSION@">3.6</token>
```

Comment:
If this is just a few lines, you could move this macro directly into the main file.

Reply:
Done!